ios/Podfile

```diff
@@ -57,4 +60,14 @@ end
 target 'GoldWallet Dev' do
   sharedPods
   pod 'react-native-config', :path => '../node_modules/react-native-config'
+
+    # Enables Flipper.
+  #
+  # Note that if you have use_frameworks! enabled, Flipper will not work and
+  # you should disable the next line.
+  use_flipper!()
```

Comment:
yee same as top we need this flipper ? suspect lots problems in ours CI/CD

Reply:
answered above